lib/utilities.bash

```diff
@@ -1,27 +1,30 @@
-#!/usr/bin/env bash
+# shellcheck shell=bash
 #
 # A collection of reusable functions.
 
 ###########################################################################
 # Generic utilies
 ###########################################################################
 
-_bash-it-get-component-name-from-path() {
-  # filename without path
-  filename=${1##*/}
-  # filename without path or priority
-  filename=${filename##*---}
-  # filename without path, priority or extension
-  echo ${filename%.*.bash}
+function _bash-it-get-component-name-from-path() {
+	local filename
+	# filename without path
+	filename="${1##*/}"
+	# filename without path or priority
+	filename="${filename##*${BASH_IT_LOAD_PRIORITY_SEPARATOR?}}"
```

Comment:
seems like this generates SC2295 warning, somehow this slipped here @gaelicWizard